ai21/errors.py

```diff
@@ -94,6 +94,12 @@ def __init__(self, key: str):
         super().__init__(message)
 
 
+class CredentialsError(AI21Error):
+    def __init__(self, error_message: str):
+        message = f"Could not get default GCP credentials: {error_message}"
```

Comment:
This should maybe be a bit more generic and accept, `provider` as an arg. This would we could add it for AWS, Azure, etc

Reply:
Noice